R/sun-methods.R

```diff
@@ -20,13 +28,33 @@
     list(crds=crds, dateTime=dateTime)
 }
 
-###_ + crepuscule methods
-if (!isGeneric("crepuscule")) {
-    setGeneric("crepuscule", function(crds, dateTime, ...) {
-        standardGeneric("crepuscule")
-    })
-}
+#' @title Compute crepuscular time
+#' @description Calculates the crepuscular time, i.e., the time of dawn or dusk, 
+#' at a specific geographical location and time. Methods are available for different 
+#' object types with geographical coordinates, including:
+#' * "sf": an object of class "sf".
+#' * "matrix": a matrix of coordinates.
+#' * "SpatialPoints": an object of class "SpatialPoints".
+#'
+#' @param crds This represents geographical coordinates. It can be an object of
+#' class "sf", "matrix", or "SpatialPoints".
```

Comment:
classes are usually referenced with back ticks `sf`, `matrix` etc., which renders them in courier font.

Reply:
i haven't indicated all of these in my review, there are several instances